cli/src/main/java/org/apache/iotdb/tool/ExportCsv.java

```diff
@@ -65,13 +63,23 @@
   private static final String SQL_FILE_ARGS = "s";
   private static final String SQL_FILE_NAME = "sqlfile";
 
+  private static final String DATA_TYPE_ARGS = "datatype";
+  private static final String DATA_TYPE_NAME = "datatype";
+
+  private static final String QUERY_COMMAND_ARGS = "q";
+  private static final String QUERY_COMMAND_NAME = "queryCommand";
+
   private static final String TSFILEDB_CLI_PREFIX = "ExportCsv";
 
   private static final String DUMP_FILE_NAME_DEFAULT = "dump";
   private static String targetFile = DUMP_FILE_NAME_DEFAULT;
 
   private static String targetDirectory;
 
+  private static Boolean dataType;
```

Comment:
What is the meaning of dataType? why is a boolean?

Reply:
It' a parameter to specify wheter print the data type if timeseries in the head line of CSV file.

Reply:
How about choice another name? The current one is hard to understand...

Reply:
Ok, I got it.

Reply:
How about "isDataTypePrinted"?

Reply:
Use needDataTypePrinted?

Reply:
Ok